mqtt-client-Electron/index.html

```diff
@@ -53,7 +66,10 @@ <h1 class="card-header-title">
                 </div>
               </div>
               <div class="column">
-                <div></div>
+                <label class="label">Path</label>
+                <div class="control">
+                  <input class="input" name="path" type="text" value="/mqtt">
+                </div>
```

Comment:
This feature is only supported on MQTT over WebSocket and should be handled with special care.

Reply:
```javascript
let connectUrl = `${protocol.value}://${host.value}:${port.value}`
  if (protocol.value === 'ws' || protocol.value === 'wss') {
    connectUrl = connectUrl + path.value
}
```

Only when the ws and wss protocols are selected, the path will be added to the connectUrl.
Is this ok?

Reply:
hide path form item, if protocol is not ws or wss?

Reply:
YES. It's not working for MQTT/TCP, you can ref the MQTT X